testApi/src/main/scala/africa/absa/testing/scapi/rest/response/action/AssertionResponseAction.scala

```diff
@@ -211,13 +287,11 @@ object AssertionResponseAction extends ResponseActions {
    * @return A Try[Unit] that is successful if the response's status code matches the expected code, and contains an exception otherwise.
    * @throws AssertionException if the response's status code does not match the expected code.
    */
-  private def assertStatusCodeEquals(response: Response, expectedCode: String): Try[Unit] = {
-    Try {
-      val iExpectedCode: Int = expectedCode.toInt
+  private def assertStatusCodeEquals(response: Response, expectedCode: String): Try[Unit] = Try {
+    val iExpectedCode: Int = expectedCode.toInt
```

Comment:
```suggestion
    val expectedCodeInt: Int = expectedCode.toInt
```

```suggestion
    val expectedCodeAsInt: Int = expectedCode.toInt
```

or just rename the input param to e.g. `expectedCodeStr` to have just `val expectedCode: Int` here.

otherwise the `i` prefix looks like `input` or `interface`, not very scala-like. Just a feeling, cannot find a proper reference :(

Reply:
Fixed in commit 04e60c441ad8b397652ec2840da18f96300f1de6.